compliance/component.yaml

```diff
@@ -0,0 +1,237 @@
+schema_version: 3.0.0
+name: FedRAMP Dashboard
+documentation_complete: false
+references:
+- name: Amazon Web Services' S3 Authorized URLs
+  path: http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html
+  type: URL
+- name: New Relic Application Monitoring
+  path: https://newrelic.com/application-monitoring
+  type: URL
+- name: Repository's Github
+  path: https://github.com/18F/fedramp-dashboard
+  type: URL
+- name: JSHint, Python Linting Tool
+  path: http://jshint.com/
+  type: URL
+- name: Code Climate, Static Analysis
+  path: https://codeclimate.com/github/18F/fedramp-dashboard
+  type: URL
+- name: OWASP's ZAP
+  path: https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project
+  type: URL
+- name: Nessus
+  path: http://www.tenable.com/products/nessus-vulnerability-scanner
+  type: URL
+satisfies:
+- standard_key: NIST-800-53
+  control_key: AC-2   # Account Management
+  narrative:
+    - text: >
+        Within our application (see cloud.gov for lower-level controls), user
+        accounts are not created or managed.
+- standard_key: NIST-800-53
+  control_key: AC-3   # Access Enforcement
+  narrative:
+    - text: >
+        The application's functionality, including "read" access to
+        FedRAMP data, is meant to be
+        accessed by the general public. That said, we do have certain
+        restrictions on user abilities.
+- standard_key: NIST-800-53
+  control_key: AC-6   # Least Privilege
+  narrative:
+    - text: >
+        At the application level (see cloud.gov for lower-level controls), users
+        are permitted access to public data.
+  covered_by:
+    - verification_key: aws-s3-url
+- standard_key: NIST-800-53
+  control_key: AU-2   # Audit Events
+  narrative:
+    - text: >
+        Cloud.gov logs requests, failures, warnings, etc. emitted by the
+        application.
```

Comment:
Looks like we might want to also mention that we are state "We also utilize New Relic's Browser monitoring."